cterasdk/edge/aio.py

```diff
@@ -6,6 +6,15 @@ class AIO(BaseCommand):
     """
     Gateway AIO APIs
     """
+    def is_enabled(self):
+        """
+        Is AIO enabled
+
+        :return: True is AIO is enabled, else False
+        :rtype: bool
+        """
+        cifs = self._gateway.get('/config/fileservices/cifs')
```

Comment:
It's a bit tricky. The status of AIO is determined by 3 values:
1) aioReadThreshold
2) aioWriteThreshold
3) robustMutexes
I recommend comparing all 3 and return a Boolean of that comparison

Reply:
@saimonation What should the comparison for the Threshold be? 1 (like we set in the enable API) or anything that is not 0?

Reply:
The code enforces values greater than 0. So check for values greater than 0.

Reply:
Done